# Define the goal
casannis_walking/Obstacles platform
---
# Define the result
bool demo_result    # True or false result for execution or not
---
# Define a feedback message
std_msgs/String demo_feedback
